E/Elfutils/build_tarballs.jl

```diff
@@ -27,6 +27,10 @@ if [[ ${target} = *-musl* ]] ; then
     mkdir -p $prefix/include/sys
     # Skip warning macro at top of file
     tail -n +2 /usr/include/sys/cdefs.h >$prefix/include/sys/cdefs.h
+    mkdir $WORKSPACE/bin
+    export PATH=$PATH:$WORKSPACE/bin
+    : >$WORKSPACE/bin/autopoint
+    chmod a+x $WORKSPACE/bin/autopoint
```

Comment:
Isn't it easier to just install `autopoint`
https://github.com/JuliaPackaging/Yggdrasil/blob/6ec5f51309bf3509ae31a2642f8a124d9c65de98/Z/zbar/build_tarballs.jl#L16-L17
?

Reply:
Done.